src/aics-image-viewer/shared/utils/playControls.ts

```diff
@@ -0,0 +1,80 @@
+import { AxisName } from "../types";
+
+type PlayAxisName = AxisName | "t";
+
+const PLAY_STEP_INTERVAL = 125;
+
+export default class PlayControls {
+  playingAxis: PlayAxisName | null = null;
+  playWaitingForLoad = false;
+  playHolding = false;
```

Comment:
I like that there's an explicit holding state. In colorizer I'm tracking this outside my timing controller and pausing/playing,  which is a little messier.